app/components/primer/alpha/select_panel_element.ts

```diff
@@ -84,6 +84,7 @@ export class SelectPanelElement extends HTMLElement {
   #selectedItems: Map<string, SelectedItem> = new Map()
   #loadingDelayTimeoutId: number | null = null
   #loadingAnnouncementTimeoutId: number | null = null
+  #userHasMadeSelection: boolean = false
```

Comment:
For multiselect, If s server returns a subset of items (say 10 out of 100 and all are selected) then when we filter by input and get a new set of items some items will not be selected that should be selected.

Reply:
Good catch!